dokka-subprojects/plugin-base/src/test/kotlin/model/MultiLanguageInheritanceTest.kt

```diff
@@ -26,6 +27,48 @@ class MultiLanguageInheritanceTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun `should not try to find javadoc for non JVM source set`() {
```

Comment:
BTW, may be can move this test to `analysis-kotlin-api`?

Reply:
There's no API for multiplatform projects yet, unfortunately :( I didn't add it because in case something changed in the underlying framework (TestProject, TestFile, TestData, etc) it would take more time and effort to update all implementation, so for now it supports only JVM projects. If it proves to be convenient, we'll add multiplatform and move all tests from dokka-base into it